src/mongo/tree/MongoCollectionTreeItem.ts

```diff
@@ -189,6 +192,11 @@ export class MongoCollectionTreeItem implements IAzureParentTreeItem {
 			});
 	}
 
+	private async count(args?: any): Promise<string> {
+		const count = await this.collection.count(args);
+		return String(count);
```

Comment:
Let's do `this.stringify({ count })` instead of `String(count)` so that we're consistent with other commands